docs/make.jl

```diff
@@ -36,16 +36,16 @@ PAGES = [
     "Documentation" => "index.md",
     "What is a surrogate?" => "man/whatisasurrogate.md",
     "Example applications" => [
-        "Shuffle-based" => "constrained/randomshuffle.md",
-        "Fourier-based" => "constrained/fourier_surrogates.md",
-        "Amplitude-adjusted FT" => "constrained/amplitude_adjusted.md",
-        "Pseudo-periodic" => "constrained/pps.md",
-        "Wavelet-based" => "constrained/wls.md",
-        "Pseudo-periodic twin" => "constrained/ppts.md",
-        "Wavelet-based" => "constrained/wls.md",
-        "Multidimensional surrogates" => "constrained/multidim.md",
-        "Surrogates for irregular timeseries" => "constrained/irregular_surrogates.md",
-        "Surrogates for nonstationary timeseries" => "constrained/nonstationary.md"
+        "Shuffle-based" => "methods/randomshuffle.md",
+        "Fourier-based" => "methods/fourier_surrogates.md",
+        "Amplitude-adjusted FT" => "methods/amplitude_adjusted.md",
+        "Pseudo-periodic" => "methods/pps.md",
```

Comment:
I think you need to refresh to current master or something like that; these commits really shouldn't be here

Reply:
Not sure what happened here. Somehow, I didn't get the latest commits to master locally when fetching. Will clean up this.